celery_utils/migrations/0003_rename_failedtask_task_name_exc_idx_failedtask_task_exc.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.11 on 2025-04-09 02:33
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('celery_utils', '0002_chordable_django_backend'),
+    ]
+
+    operations = [
+        migrations.RenameIndex(
+            model_name='failedtask',
+            new_name='idx_failedtask_task_exc',
+            old_fields=('task_name', 'exc'),
+        ),
+    ]
```

Comment:
`ALTER INDEX "celery_utils_failedtask_task_name_exc_efb8c9be_idx" RENAME TO "idx_failedtask_task_exc";`

@awais786 here is the query it is generating for postgresql.

Reply:
Nice, this is also a low cost metadata only query so it should be safe to run on even large tables.

Reply:
agreed!